demo/SnowplowDemo/Analytics/Tracker.swift

```diff
@@ -0,0 +1,19 @@
+import Foundation
+import SnowplowTracker
+
+public class Tracker {
+    public static let shared = Tracker()
+    private let tracker: TrackerController
+
+    init() {
+        tracker = Snowplow.createTracker(namespace: "DemoAppExtensionNamespace", endpoint: "acme.fake.com", method: .post)
```

Comment:
Initializing the tracker with the default configuration activates a lot of autotracking functionality that results in tracking events that are not really interesting within the extension setting. Could you please provide a custom `TrackerConfiguration` that deactivates the autotracking? In my trials, I [used a tracker config that you can find here](https://github.com/snowplow-incubator/snowplow-objc-tracker-examples/blob/f48d373272d7256de9610f37d0087756b5f897f0/demo/SnowplowDemo/SnowPlowDemoShareExtension/Tracker.swift#L9-L13).